### ISSUE TITLE
Error in annoteBandOncoHeat    and       Error in data.frame

### ISSUE THREAD

**Issue by Crick0509** [AntonioDeFalco/SCEVAN#46]
Thank you for your contribution to the bioinformation,I  think SCEVAN is a gospel for many beginner.But I meet some problems when I run it.

the first problem is:
[1] "found 3 subclones"
percentage_cells_subsclone_1 percentage_cells_subsclone_2 percentage_cells_subsclone_3 
                   0.5175202                    0.2318059                    0.2506739 
[1] "Segmentation of subclone :  1"
[1] "Segmentation of subclone :  2"
[1] "Segmentation of subclone :  3"
$`_subclone1`
    Chr     Start       End Alteration segm.mean
15    2 111898479 119438520          2  0.180712
31    4  57030773  75941051          2  0.317688
63    8  81701334 102412841          2  0.177557
143  22  50185915  50628173         -2 -0.199268
59    7 140084746 158830253         -1 -0.122816

$`_subclone2`
   Chr     Start       End Alteration segm.mean
26   6  30676389  33416453         -2 -0.261065
56  11    167784   8920084         -2 -0.139116
97  19  12796820  49526333         -2 -0.194117
64  12 110434825 120201235         -1 -0.102395
71  14  49893092  64533690          1  0.119032
86  17   5419641   8290092         -1 -0.122950

$`_subclone3`
   Chr     Start       End Alteration segm.mean
6    1 151156664 153946696         -2 -0.183453
99  19  45488777  49658642         -2 -0.167078

$`_shareSubclone`
        Chr    Start      End Alteration segm.mean sh_sub Mean
_share1   1 19303965 24673267         -2 -0.140455    1-2    0

### **_Error in annoteBandOncoHeat(res_proc$count_mtx_annot, diffSubcl, res_subclones$n_subclones,  : 
  (converted from warning) NAs introduced by coercion_**

####################################################################################################
but when I run another count_mtx,another problem occured:
[1] "found 5 subclones"
percentage_cells_subsclone_1 percentage_cells_subsclone_2 percentage_cells_subsclone_3 percentage_cells_subsclone_4 
                   0.2212121                    0.2252525                    0.2868687                    0.1686869 
percentage_cells_subsclone_5 
                   0.0979798 
[1] "Segmentation of subclone :  1"
[1] "Segmentation of subclone :  2"
[1] "Segmentation of subclone :  3"
[1] "Segmentation of subclone :  4"
[1] "Segmentation of subclone :  5"
Error in data.frame(..., check.names = FALSE) : 
  (converted from warning) 

this is my running code,and 'epithelial' is my seurat result:
library(devtools)
install_github("miccec/yaGST")
install_github("AntonioDeFalco/SCEVAN")

library(SCEVAN)
library(Seurat)
suppressPackageStartupMessages(library(ggtree)) 
count_mtx = epithelial@assays$RNA@counts
count_mtx[1:4,1:4]
phe=epithelial@meta.data
head(phe)
colnames(phe)

count_mtx[1:4,1:4]
results <- pipelineCNA(count_mtx)

I don't know how can I do for it and looking forward to your help!

**Comment by AntonioDeFalco** [comment]
> but when I run another count_mtx,another problem occured:
[1] "found 5 subclones"
percentage_cells_subsclone_1 percentage_cells_subsclone_2 percentage_cells_subsclone_3 percentage_cells_subsclone_4
0.2212121 0.2252525 0.2868687 0.1686869
percentage_cells_subsclone_5
0.0979798
[1] "Segmentation of subclone : 1"
[1] "Segmentation of subclone : 2"
[1] "Segmentation of subclone : 3"
[1] "Segmentation of subclone : 4"
[1] "Segmentation of subclone : 5"
Error in data.frame(..., check.names = FALSE) :
(converted from warning)

Hello @Crick0509 regarding this error, have you tried reducing the number of cores used in parallel as suggested in previous issues? #43 #38 

As for the other error, I am trying to understand what it could be due to.

Regards.

**Comment by AntonioDeFalco** [comment]
Duplicate of https://github.com/AntonioDeFalco/SCEVAN/issues/43 https://github.com/AntonioDeFalco/SCEVAN/issues/38
